src/setup/mouse.c

```diff
@@ -106,6 +107,7 @@ static void ConfigExtraButtons(TXT_UNCAST_ARG(widget), TXT_UNCAST_ARG(unused))
     AddMouseControl(buttons_table, "Previous weapon", &mousebprevweapon);
     AddMouseControl(buttons_table, "Strafe on", &mousebstrafe);
     AddMouseControl(buttons_table, "Next weapon", &mousebnextweapon);
+    AddMouseControl(buttons_table, "Run on", &mousebspeed);
```

Comment:
In the keyboard and joystick dialogs this is called "Speed on" which is the original label from setup.exe. However, most people call it "running" and we even have the label "Always run". Regardless we should be consistent. I suggest changing the label to just "Run" (not "Run on") in all three - what do you think?

(Why not "Run on"? "Strafe on" is only so named to distinguish it from "Strafe left" and "Strafe right")

Reply:
No qualms here! I actually originally called it "Run", but renamed it to be consistent with "Strafe on" since it was another modifier button, and I figured that was just convention.

But if it's not, I agree that reads much nicer!

Reply:
Thanks - I was hoping you could fix the keyboard/joystick dialogs while you're here?

Reply:
Sure! Like this?

Reply:
Perfect, thank you.